api/lib/mno_enterprise/concerns/controllers/jpi/v1/marketplace_controller.rb

```diff
@@ -30,6 +30,19 @@ def show
     @app = MnoEnterprise::App.find(params[:id])
   end
 
+  # GET /mnoe/jpi/v1/marketplace/:id/app_comments
+  def app_reviews
+    res = MnoEnterprise::AppReview.where(reviewable_id: params[:id])
+    res.limit(params[:limit]) if params[:limit]
+    res.skip(params[:offset]) if params[:offset]
+    res.order_by(params[:order_by]) if params[:order_by]
+    res.where(params[:where]) if params[:where]
+    @app_reviews = res.all.fetch
+    response.headers['X-Total-Count'] = @app_reviews.metadata[:pagination][:count]
+    return render json: "could not find Reviews for app #{params[:id]}", status: :not_found unless @app_reviews
```

Comment:
I added the empty array management in the front end, I thought it could be useful to have it in the backend too.. I see it wasn't 😅